widgetastic_manageiq/__init__.py

```diff
@@ -5127,3 +5127,41 @@ def click_body_action(self):
 
     def click(self):
         return self.click_title()
+
+
+class SnapshotMemorySwitch(Widget):
+    """ This replaces the old-style checkbox when working with snapshot memory.
+
+    At first glance on the page it looks like a BootstrapSwitch but it really isn't.
+    Basic switch controls are implemented here along with fill and read methods,
+    so this should be usable as any other widget.
+    """
+
+    ROOT = '//div[contains(@class, "bootstrap-switch-snap_memory")]'
+    ON_LOCATOR = './/div[contains(@class, "bootstrap-switch-on")]'
+
+    def switch(self):
```

Comment:
I don't think there's a real need for `switch`, just `switch_on` and `switch_off`.  `switch` itself, in that context, becomes a bit ambiguous. 

Please remove this method, and include `ClickableMixin` in your class declaration.  The `switch_on` and `switch_off` methods can then call `self.click()`, which I think is a more explicit action than `switch` alone.